logic.py

```diff
@@ -121,17 +122,23 @@ def handle_proofreader_file(request, assignment, article):
             )
 
 
-def new_typesetting_round(article, rounds, user):
+def new_typesetting_round(article, rounds, request):
```

Comment:
Why do we make this change? the function requires the user, not the request

Reply:
`notify.event_typesetting_cancelled` called later needs the request object.